src/styles/pages/style.scss

```diff
@@ -0,0 +1,844 @@
+  width: 89px;
+  /* Fixed missing unit */
+  height: 40px;
+  gap: 8px;
+  padding-top: 8px;
+  padding-right: 20px;
+  padding-bottom: 8px;
+  padding-left: 20px;
+  border-bottom-width: 2px;
+  list-style: none;
+  display: flex;
+  margin: 0;
+  padding: 0;
+}
+
+.main-nav__link {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-weight: 600;
+  font-size: 16px;
+  color: #F5F8FB;
+  text-decoration: none;
+  padding: 10px 15px;
+  width: 49px;
+  height: 24px;
+  line-height: 24px;
+  letter-spacing: 0px;
+  text-align: center;
+  vertical-align: middle;
+  color: #ffffff;
+  font-weight: 500;
+}
+
+.main-nav__link:hover {
+  text-decoration: underline;
+}
+
+.main-nav__list-item {
+  margin-right: 20px;
+}
+
+/* .main-nav__list-item {
+  font-family: 'Poppins', sans-serif;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 1.5;
+  color: #F5F8FB;
+  margin-right: 20px;
+} DO NOT DUPLICATE STYLES. IF YOU DO NOT HAVE TEXT CONTENT INSIDE THE TAG/SELECTOR */
+
+
+.search {
+  padding: 5px;
+  margin-right: 20px;
+  border: 1px solid #333;
+  border-radius: 4px;
+  background-color: #2c2c2c;
+  color: #ffffff;
+}
+
+
+/* Hero Banner */
+.banner {
+  width: 866.500732421875px;
+  height: 362px;
+  background: radial-gradient(44.91% 42.19% at 22% -4%, rgba(2, 101, 220, 0.5), rgba(2, 101, 220, 0) 100%), rgb(13, 13, 21);
+}
+
+.banner__heading {
+  font-size: 48px;
+  margin-bottom: 20px;
+  line-height: 1.2;
+}
+
+.banner__highlight {
+  color: #0265DC;
+}
+
+.banner__text {
+  font-family: 'Poppins', Arial, sans-serif;
+  font-size: 18px;
+  margin-bottom: 30px;
+  line-height: 1.5;
+}
+
+.banner__stats {
+  display: block;
+  width: 100%;
+  max-width: 800px;
+}
+
+.banner__stat-label {
+  font-family: 'Poppins', Arial, sans-serif;
+  width: 95px;
+  height: 24px;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  letter-spacing: 0px;
+  vertical-align: bottom;
+}
+
+.banner__stat-value {
+  box-shadow: 0px 4px 13px 0px rgba(255, 255, 255, 0.4);
+  font-size: 52px;
+  line-height: 62px;
+  letter-spacing: -0.5px;
+  vertical-align: bottom;
+}
+
+.banner__image {
+  width: 533.499267578125px;
+  height: 641.8035888671875px;
+  top: -0.9px;
+  left: 1030.5px;
+}
+
+.banner__button-group {
+  width: 866.500732421875px;
+  height: 48px;
+  gap: 16px;
+}
+
+.banner__title {
+  background-image: url(src\images\static\our-blog-background.webp);
+}
+
+/* Button Styles */
+.button {
+  display: inline-block;
+  font-family: 'Poppins';
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 1.5;
+  text-decoration: none;
+  padding: 8px 20px;
+  color: white;
+}
+
+.button--primary {
+  background: #0265DC;
+  background-color: #007bff;
+  color: #ffffff;
+}
+
+.button--secondary {
+  background: rgba(208, 215, 251, 0);
+  border: 2px solid rgba(208, 215, 251, 0.17);
+  margin: 0 10px;
+  padding: 10px 20px;
+  background-color: #333;
+  color: #ffffff;
+}
+
+.button--secondary:hover,
+.button--primary:hover {
+  opacity: 0.8;
```

Comment:
Hover state behavior: The hover effect should change the background color, not opacity. Update the style accordingly.